pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/SchemaDefinitionBuilderImpl.java

```diff
@@ -35,7 +35,9 @@
 public class SchemaDefinitionBuilderImpl<T> implements SchemaDefinitionBuilder<T> {
 
     public static final String ALWAYS_ALLOW_NULL = "__alwaysAllowNull";
+    public static final String ALWAYS_ALLOW_NULL_NO_UNDERLINED = "alwaysAllowNull";
```

Comment:
Can you explain why do you introduce two new settings without underscores?

Reply:
Why can't we use the existing settings?

Reply:
Already use existing settings。This may cause some parameters to be underscores, some not
@sijie